### ISSUE TITLE
Resource restart = server crash

### ISSUE THREAD

**Issue by Lekenthu** [altmp/altv-issues#2382]
### Description of the problem

Since the latest release build, the whole server crashes on any resource restart. Previous releases had no such problem.

### Reproduction steps

Restart any resource.

### Expected behaviour

- 

### Operating system

Windows 11

### Version

alt:V build #16.2.26, branch release

### Confirmation of issue's presence

- [X] *By submitting this ticket, I affirm that I have verified the presence of this issue on the latest RC (Release Candidate) version available at the time of writing this ticket.*

**Comment by co-pas** [comment]
I'm really trying to understand why

**Comment by BluePeer** [comment]
until it is fixed
you can use a Workaround

```

            string oldresourcefile = $"./cache/{Alt.Resource.Name}.resource";
            if (File.Exists(oldresourcefile)) File.Delete(oldresourcefile);
```
In C# i use this on the end of onstop for the serverside resource, to prevent the crash related to the existent .resource file of the restarting resource if you wan't to restart client resourcen, you need to delete the file before too.
(example für none container mode)

**Comment by Doxoh** [comment]
should be fixed in `16.2.28-rc.1`

**Comment by BluePeer** [comment]
confirmed
